js/validate.js

```diff
@@ -0,0 +1,35 @@
+const form = document.querySelector('#form');
+const EMAIL_INVALID = 'Sorry!, form was not submitted. Your email should be in lowercase e.g abc@gmail.com';
+
+function displayMessage(input, message, type) {
+  document.querySelector('small').innerText = message;
+  input.className = type ? 'success' : 'error';
+  return type;
+}
+
+function showError(input, message) {
+  return displayMessage(input, message, false);
+}
+
+function showSuccess(input) {
+  return displayMessage(input, '', true);
+}
+
+function validateEmail(input, invalidMsg) {
+  const email = input.value.trim();
+  if (email !== email.toLowerCase()) {
+    return showError(input, invalidMsg);
```

Comment:
Please ensure that the error message should show up close to the `submit button`. And that the error message fits the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins).

Reply:
Thank you for the feedback, but I can't seem to find on Figma how the error message should be displayed such as the color,  position, typo . Pls can you give me more clarity?

Reply:
Please ensure the error message is close to the submit button.

Reply:
noted. Thanks